src/schedule-and-details/introducing-section/course-card-image/index.jsx

```diff
@@ -0,0 +1,148 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  injectIntl,
+  intlShape,
+  FormattedMessage,
+} from '@edx/frontend-platform/i18n';
+import { FileUpload as FileUploadIcon } from '@edx/paragon/icons';
+import {
+  Form,
+  Dropzone,
+  Image,
+  Hyperlink,
+  Card,
+  Icon,
+  IconButton,
+} from '@edx/paragon';
+
+import { updateAssetsQuery } from '../../data/thunks';
+import { getUploadedAssets } from '../../data/selectors';
+import { assetPathRoot, assetsUrl } from '../utils';
+import messages from './messages';
+
+const CourseCardImage = ({
+  intl,
+  courseId,
+  courseImageAssetPath,
+  onChange,
+}) => {
+  const dispatch = useDispatch();
+  const uploadedAssets = useSelector(getUploadedAssets);
+  const imageAbsolutePath = assetPathRoot(`/${_.last(courseImageAssetPath.split('/'))}`);
```

Comment:
why don't you use js util URL, which does all the things with handling extra slashes?
for example
```
const url = new URL(courseImageAssetPath, getConfig().STUDIO_BASE_URL)
url.href
```

Reply:
It's a good idea, thx